build-system/tasks/presubmit-checks.js

```diff
@@ -104,6 +104,14 @@ const forbiddenTerms = {
       'build-system/app.js',
       'build-system/amp4test.js',
       'build-system/check-package-manager.js',
+      'build-system/pr-check/build.js',
```

Comment:
I think `build-target.js` is missing from this list. (While you're at it, pluralize the filename to `build-targets.js`.)